src/views/Settings.tsx

```diff
@@ -1,227 +1,48 @@
-                                            loading={loading}
-                                            defaultValue={"Enabled"}
-                                        >
-                                            <Form.Item
-                                                label="Peer login expiration"
-                                                name="peer_login_expiration_enabled"
-                                                tooltip="Peer login expiration allows to periodically request re-authentication of peers that were added with the SSO login. You can disable the expiration per peer in the peers tab."
-                                                //rules={[{validator: selectValidatorEmptyStrings}]}
-                                            >
-                                                <Radio.Group
-                                                    options={[{label: 'Enabled', value: true}, {
-                                                        label: 'Disabled',
-                                                        value: false
-                                                    }]}
-                                                    optionType="button"
-                                                    buttonStyle="solid"
-                                                    onChange={function (e) {
-                                                        setFormPeerExpirationEnabled(e.target.value)
-                                                    }}
-                                                />
-                                            </Form.Item>
-                                            <Form.Item name="peer_login_expiration_formatted"
-                                                       label="Peer login expires in"
-                                                       tooltip="Time after which every peer added with SSO login will require re-authentication."
-                                                       rules={[{validator: checkExpiresIn}]}>
-                                                <ExpiresInInput
-                                                    disabled={!formPeerExpirationEnabled}
-                                                    options={Array.of({key: "hour", title: "Hours"}, {
-                                                        key: "day",
-                                                        title: "Days"
-                                                    })
-                                                    }/>
-                                            </Form.Item>
-                                            <Form.Item>
-                                                <Button icon={<QuestionCircleFilled/>} type="link" target="_blank"
-                                                        href="https://netbird.io/docs/how-to-guides/periodic-authentication">Learn more about login expiration</Button>
```

Comment:
I think that it makes sense to move it more to the left

Reply:
First that's a delete part. But I moved it someplace else. I can change the indention but that somehow breaks the level, I don't know if that is right, every code formatter will move it back again